iron-ajax.html

```diff
@@ -277,8 +277,19 @@
         value = this.params[param];
         param = window.encodeURIComponent(param);
 
-        if (value !== null) {
-          param += '=' + window.encodeURIComponent(value);
+        //if this parameter is an array
+        if( Object.prototype.toString.call( value ) === '[object Array]' ){
+          //we create the queryString differently
+          var URIEncodedArrayItems = [];
+          param = window.encodeURIComponent(param + "[]");
```

Comment:
This is a double encodeURIComponent on the parameter name. (it's also called on 278).

This code (before your change) conflates the final encoded param with the parameter name keys in this.params. It's probably worth it to separate out a paramName local variable here for clarity.